Lib/test/test_weakref.py

```diff
@@ -250,7 +248,7 @@ def test_basic_proxy(self):
         L2 = collections.UserList(L)
         p2 = weakref.proxy(L2)
         self.assertEqual(p, p2)
-        ## self.assertEqual(repr(L2), repr(p2))
```

Comment:
This line is not expected to be uncommented. (it doesn't have any `RUSTPYTHON` tag)